ui/admin/app/components/form/onboarding/index.hbs

```diff
@@ -0,0 +1,73 @@
+<Rose::Form
+  class='full-width'
+  @cancel={{@cancel}}
+  @onSubmit={{fn @submit this.targetAddress this.targetPort}}
+>
+  <div class='create-resources-form'>
+    <Hds::Form::TextInput::Field
+      name='targetAddress'
+      @type='text'
+      @value={{this.targetAddress}}
+      @isRequired={{true}}
+      @isInvalid={{this.targetAddressInValid}}
+      {{on 'input' (set-from-event this 'targetAddress')}}
+      {{on 'focusout' this.isTargetAddressInValid}}
```

Comment:
Not sure what other's thoughts are but it feels a little weird to have it on the `focusout` event, I wonder if it might be better to validate when trying to save to mimic how our validation works on forms normally?

Reply:
@ZedLi 
Initially I wanted to do it at the same method we submit but encounter some scope limitations. Maybe I didn't though about it properly, let's catch up about this one during pod standup zoom call.